overlord/configstate/configcore/export_test.go

```diff
@@ -35,13 +32,40 @@ var (
 	SwitchDisableService = switchDisableService
 	UpdateKeyValueStream = updateKeyValueStream
 	AddFSOnlyHandler     = addFSOnlyHandler
-	AddWithStateHandler  = addWithStateHandler
 	FilesystemOnlyApply  = filesystemOnlyApply
-	StoreReachable       = storeReachable
 )
 
 type PlainCoreConfig = plainCoreConfig
 
+type MockConf interface {
```

Comment:
do we need this one?

Reply:
Removed (somewhat I overlooked removing it...)